single_tech_samples/databricks/sample4_ci_cd/README.md

```diff
@@ -1 +1,256 @@
-# Readme
+# Azure Databricks CI/CD template
+
+## Contents
+
+- [1. Solution Overview](#1-solution-overview)
+  - [1.1. Scope](#11-scope)
+  - [1.2. Architecture](#12-architecture)
+  - [1.3. Technologies used](#13-technologies-used)
+- [2. How to use this template](#2-how-to-use-this-sample)
+  - [2.1. Prerequisites](#21-prerequisites)
+    - [2.1.1 Software Prerequisites](#211-software-prerequisites)
+  - [2.2. Setup and deployment](#22-setup-and-deployment)
+  - [2.3. Deployed Resources](#23-deployed-resources)
+  - [2.4. Sample project Structure](#24-sample-project-structure)
+    - [2.4.1 multiple notebooks](#241-multiple-notebooks)
+    - [2.4.2 notebook by pyspark API](#242-notebook-by-pyspark-API)
+    - [2.4.3 notebook by sparksql](#243-notebook-by-sparksql)
+    - [2.4.4 notebook plus python module](#244-notebook-plus-python-module)
+    - [2.4.5 python spark jo](#245-python-spark-job)
+
+## 1. Solution Overview
+
+When build a project in databricks, we can start from some notebooks and implement the business logic in python. Before go-production, we need to create CI/CD pipelines. To reduce the effort of build CI/CD pipelines, we build this git repository as template including sample noteboks and unit testing plus Azure DevOps yaml files as CI/CD pipelines.
```

Comment:
Do we want to say Python or SparkSQL as we also have a sample for CI and UT on SparkSQL